### ISSUE TITLE
Can't connect to my FTP

### ISSUE THREAD

**Issue by Hedronmx** [Alecaddd/taxi#42]
I tried connecting to my ftp server but I can't, I am using the same configuration in Filezilla and its working.

Can we have some sort of error notice, I don't know what's wrong. It just shows Can't connect.

**Comment by Alecaddd** [comment]
Yes, this app needs some serious UX love and proper error handling.
I'll work on it as soon as I find some free time.
Thanks for using Taxi

**Comment by Hedronmx** [comment]
Appreciate it!! Love to have a really good FTP client :D
